src/descriptor/policy.rs

```diff
@@ -108,39 +108,60 @@ impl PKOrF {
 #[serde(tag = "type", rename_all = "UPPERCASE")]
 pub enum SatisfiableItem {
```

Comment:
While you are fixing docs in this module, the comment on this type has a grammar typo :)
```
/// An item that need to be satisfied
```
Should be
```
/// An item that needs to be satisfied
```

Reply:
👍 Fixed